src/Notifier.php

```diff
@@ -41,39 +41,46 @@ public function addFilter($filter)
         $this->filters[] = $filter;
     }
 
-    /**
-     * Builds Airbrake notice from exception.
-     *
-     * @param \Exception $exc Exception or class that implements similar interface.
-     */
-    public function buildNotice($exc)
+    private function backtrace($exc)
     {
         $backtrace = [];
         $trace = $exc->getTrace();
+        if (!$exc instanceof Errors\Base) {
```

Comment:
I will replace this with `$exc->getFile() != ''` check, which is self-documented.